torch_tvm/operators.h

```diff
@@ -3,6 +3,12 @@
 #include <tvm/relay/expr.h>
 #include <tvm/relay/op.h>
 
+#define PARAM_INDICES_convolution {1, 2}
+#define PARAM_INDICES_layer_norm {2, 3}
+#define PARAM_INDICES_linear {1, 2}
+
+#define PARAM_INDICES(op_name) PARAM_INDICES_##op_name
```

Comment:
this is a bit strange, can we inline the {1, 2} etc into the list of ops as the final param?

Reply:
See my comment above as to why I chose to do this way.